answers.md

```diff
@@ -0,0 +1,11 @@
+1: Explain the difference between RDBMS and SQL.
+An RDBMS or Relational Data Base Management System IS the actual database software. SQL is the language we use to interact with our system.
+
+2: Why do tables need a primary key?
+The primary key is important because it is what will be referenced by other tables via their foriegn keys.
+
```

Comment:
This is true, but the reason we need a `PRIMARY KEY` is to uniquely identify each record in a table.